### ISSUE TITLE
Airplay icon is visible when hiding UI

### ISSUE THREAD

**Issue by fotiDim** [JiangJuHong/FlutterPerfectVolumeControl#1]
Trying this on an iPhone 12 Pro Max. When setting `.hideUI = true` the airplay icon is visible on the top left of the status bar. The `MPVolumeView` should be moved further away.
![IMG_C2BD90D7D6E6-1](https://user-images.githubusercontent.com/2326415/119217543-2cf83400-badb-11eb-8e28-152a230b4c54.jpeg)

**Comment by JiangJuHong** [comment]
I received this issue, I will check and try to solve it. Thank you for your submission.

**Comment by JiangJuHong** [comment]
Fixed at 1.0.3

**Comment by tania628** [comment]
Thank you for the really perfect plugin!
I'm using an physical device of iPhone X and I'm having similar issues with the latest version.
In my case, a white circle is displayed on the top left.
however, simulator did not have this issue...
Is it possible to hide it completely?

![volumeControlsIssue](https://user-images.githubusercontent.com/65319868/120573233-31680980-c458-11eb-8a10-c9f803f69e1d.jpeg)

```
import 'package:flutter/material.dart';
import 'package:perfect_volume_control/perfect_volume_control.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  MyHomePage({Key key, this.title}) : super(key: key);

  final String title;

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  int _counter = 0;

  void _incrementCounter() {
    setState(() {
      _counter++;
    });
  }
  @override
  void initState() {
    super.initState();
    PerfectVolumeControl.hideUI = true;
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Text(
              'You have pushed the button this many times:',
            ),
            Text(
              '$_counter',
              style: Theme.of(context).textTheme.headline4,
            ),
          ],
        ),
      ),
      floatingActionButton: FloatingActionButton(
        onPressed: _incrementCounter,
        tooltip: 'Increment',
        child: Icon(Icons.add),
      ),
    );
  }
}
```

**Comment by JiangJuHong** [comment]
I didn't test it on the physical machine, which may be caused by the problem of changing the 'hideUi'. I will reopen this problem and try to fix it.

**Comment by JiangJuHong** [comment]
Fixed at 1.0.4

**Comment by tania628** [comment]
It is completely hidden!
Thank you for your quick response.
